base/applications/cmdutils/eventcreate/eventcreate.rc

```diff
@@ -30,3 +30,6 @@ LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
 #ifdef LANGUAGE_TR_TR
     #include "lang/tr-TR.rc"
 #endif
+#ifdef LANGUAGE_ZH_CN
+    #include "lang/zh-CN.rc"
+#endif
```

Comment:
Please add a terminating new-line after the #endif .

Reply:
fixed